flutter/lib/src/sentry_flutter.dart

```diff
@@ -119,6 +119,9 @@ mixin SentryFlutter {
     if (isIOS()) {
       options.addIntegration(loadContextsIntegration(options, _channel));
     }
+
+    // track Flutter application events
+    options.addIntegration(widgetsBindingIntegration);
```

Comment:
does this integration need to be executed before one does `runApp(MyApp)`?
if so, consider adding the integration to this list instead:
https://github.com/getsentry/sentry-dart/blob/main/flutter/lib/src/sentry_flutter.dart#L39-L41

Reply:
It doesn't necessarily *need* to be executed before `runApp(MyApp)` but it should be executed as soon as possible after `WidgetsBinding.ensureInitialized()`.

Reply:
I see, so yeah adding to the linked list will make more sense, because we'd be getting the events from the very beginning.